compile.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+RED=$(tput setaf 1)
+RESET=$(tput sgr0)
+
+# Run with `bazel` command as param (build|test)
+if [[ $1 != "build" && $1 != "test" ]]; then
+  echo "$RED""Run script with 'build' or 'test' as param$RESET"
+  exit 1
+fi
+
+if [[ ! -z "$CIRCLECI" ]]; then
+  DELETED_PACKAGES="";
+else
+  DELETED_PACKAGES="--deleted_packages $(cat .circleci/deleted_bazel_packages.txt)";
```

Comment:
Can you document what `--deleted_packages` are and why they're needed?

Reply:
@oferb done